test.php

```
@@ -0,0 +1,198 @@
+        // 可疑的魔法数字
+        if (count($result) > 10) {
+            // ...
+        }
+
+        // 返回临时变量
+        return $result;
+    }
+
+    // 函数参数过多 (Excessive Parameter List)
+    public function createUser(
+        string $firstName,
```

PHPMD (2.15.0)

Avoid unused parameters such as '$firstName'. (undefined)

(UnusedFormalParameter)